web-client/src/presenter/computeds/filingPartiesFormHelper.ts

```diff
@@ -18,11 +18,9 @@ export const filingPartiesFormHelper = (
     PARTY_TYPES,
   } = applicationContext.getConstants();
 
-  const partyValidationError =
+  const partyValidationCheckboxError =
```

Comment:
We were displaying the `validationErrors.otherFilingParty` twice, once in the general form and again in the nested child form. We now just show it in the latter.